src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -368,6 +369,7 @@ export function ConfirmationModal ({ transaction, handleClose, declineTransactio
 
   const onYes = () => {
     actionHook(actionParams)
+    if (actionHook === declineTransaction) setInboxView(VIEW['recent'])
```

Comment:
i would make this conditional on `action`, not `actionHook`. 
ie
`if (action === 'decline') setInboxView(VIEW['recent'])`

Couple of principles inform this for me
1) single source of truth. `action === 'decline'` is already how the component knows which action it's taking, so that should be our test everywhere we're asking that question
2) it's best to avoid comparing object references when possible, and especially function references.

Reply:
Solid points; I agree with this logic. :raised_hands: